CHANGELOG.md

```diff
@@ -5,8 +5,13 @@
 This release extends the planned support of the modules to OneView REST API version 800 (OneView v4.1).
 
 #### Major changes
-1. Extended support of planned modules to API800
-2. Updated library files to support Python SDK changes
+1. Extended support of planned modules to API800.
+2. Updated library files to support Python SDK changes.
+3. Modules upgraded in this release required hpOneView version 5.0.0 or above.
```

Comment:
this should be 5.3.0?

Reply:
"hpOneView"  is the python module required for this module.

Reply:
please ignore